src/components/backend-ai-scaling-group-list.ts

```diff
@@ -576,6 +772,56 @@ export default class BackendAIScalingGroupList extends BackendAIPage {
             </mwc-button>
        </div>
       </backend-ai-dialog>
+      <backend-ai-dialog id="modify-scheduler-opts-dialog" fixed backdrop persistent>
+        <span slot="title">${_t('resourceGroup.SetSchedulerOptions')}</span>
+        <div slot="content" id="modify-scheduler-opts-container">
+          <mwc-select
+          id="allowed-session-types"
+          label="allowed_session_types"
+          required>
+            <mwc-list-item style="height:auto;" value="interactive">interactive</mwc-list-item>
+            <mwc-list-item style="height:auto;" value="batch">batch</mwc-list-item>
+            <mwc-list-item style="height:auto;" value="both">both</mwc-list-item>
+          </mwc-select>
+          <mwc-textfield
```

Comment:
Three things to ask in this section:
1. I'm not sure `pending timeout` is the required value.
2. And Let's replace `_(underbar)` in the label to whitespace since is a bit unnatural.
3. Let's use change event instead of input event, since `input event` is triggered much more frequently compared to "change event". Please refer to [this link](https://stackoverflow.com/questions/17047497/difference-between-change-and-input-event-for-an-input-element).

Reply:
1. The reason why I put pending_timeout was because pending_timeout came into thisPR(https://github.com/lablup/backend.ai-manager/pull/527) as a new scheduler_opts.
2. I revise it following comment.
3. I revise it following comment.